WordPress/gutenberg
# [Latest Posts] adds author option to latest posts block

## Description
Props to @anischarolia-multidots this is simply #17163 refreshed to work with current master.
Closes: #17681

## How has this been tested?
Tested locally by:

1. Make sure you have some posts
2. Add a new page or post
3. Inser a LatestPosts block
4. Use the inspector option "Display post author" to toggle author on

## Screenshots 
<img width="1087" alt="Screenshot 2020-03-11 at 09 25 31" src="https://user-images.githubusercontent.com/107534/76392524-659a6a80-637a-11ea-8da3-8b8ee1cccf86.png">

## Types of changes
Non breaking changes adding an `InspectorControl` to the `LatestPosts` block.
